src/net.h

```diff
@@ -32,7 +32,9 @@ enum class MessageType {
     Listen,
     ValueData,
     ValueUpdate,
-    UpdateValue
+    UpdateValue,
+    RefreshValue,
+    ExpireValue
```

Comment:
could we use a single message type for the 3 ? (since it goes to the socket anyway)

Reply:
Indeed UpdateValues is definitely enough to do all the job and avoid to create one more version